File: 프로그래머스/0/181874. A 강조하기/A 강조하기.java
class Solution {
    public String solution(String myString) {
        // 정답 선언 및 초기화
        String answer = "";
        // a를 우선 숫자 0으로 변경 
        String str1 = myString.replace("a", "0");
        // 전체를 소문자로 변환
        String str2 = str1.toLowerCase();
        // 0을 대문자 A로 변경 - 이전 과정에서 소문자로 변환된 "A"도 다시 "A"로 변환
        answer = str2.replace("0", "A").replace("a", "A");
        
        // 정답 리턴
        return answer;
    }
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.py
def solution(myString):
    answer = ''
    i = 0
    
    myString = myString.lower()
    
    while i < len(myString):
        if myString[i] == 'a':
            answer += 'A'
            i += 1
        else:
            answer += myString[i]
            i += 1
            
    return answer